### DISCUSSION TITLE
Discord server for aiomysql

### DISCUSSION THREAD

**Discussion by leothewolf** [aio-libs/aiomysql#789] [Ideas]
Hey there,

Well I went through the gitter link you guys have on your github page and it seems not every1 has gitter so I though of making a discord server! I have made it and you can join it via https://discord.gg/f3ncE2w6FJ I will be more than happy to hand over the server to the main developer if you could contact me at my discord on leothewolf122#1319 or tell me through the server!

Hope you can understand the need, also yea I made a logo for aiomysql if you wanna use it! 
![Untitled-2](https://user-images.githubusercontent.com/91257665/169647897-27e1cfe7-cc6a-432b-a8f0-a2666c050243.png)

**Disclaimer :**  The logo is not all mine I have taken help from google

**Comment by Nothing4You** [comment]
Hi,

I'm afraid I don't use discord and I don't plan to start using it.

Note that you're also able to join the gitter room through matrix, without having to use gitter directly.

You're of course free to run an unofficial community there.

**Reply by leothewolf** [reply]
> Hi,
> 
> I'm afraid I don't use discord and I don't plan to start using it.
> 
> Note that you're also able to join the gitter room through matrix, without having to use gitter directly.
> 
> You're of course free to run an unofficial community there.

Thanks for approving that and the thing is on discord we have separate community for python where we have more than 347k members. Although I have started aiomysql community the thing is just so if any1 on discord faces issues he/she/they can talk there as many people use aiomysql even for discord bots! [Including me]

**and yea thanks for developing this wonderful library**

And yea if any day you join dis ill be more than happy to hand over the ownership!
